### PR TITLE
TraitsCleanupRound2

### LINKED ISSUE #1146: traits need to be pruned
accents can probably stay but anything that would effect the game for others should probably not be in here

### PR DESCRIPTION
**Pull request by Froffy025**
## About the PR

not sure why but the last time i pushed this it had my savannah changes, too. sneaky git... 
anyways. 
## Why / Balance

-fix #1146 
## Media

image

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

File: Resources/Prototypes/Traits/inconveniences.yml [hunk 1]
```diff
@@ -16,3 +16,17 @@
...
+  description: trait-frontal-lisp-desc
+  components:
+    - type: FrontalLisp
+
+- type: trait
+  id: Snoring
+  name: trait-snoring-name
+  description: trait-snoring-desc
+  components:
+    - type: Snoring
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/Traits/inconveniences.yml:32]: Just ignore `disabilities.yml` and create a new `cm_inconviences.yml` for these two traits in the CM folder, this way we don't have merge conflicts on upstream files

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Traits/inconveniences.yml (hunk 1 above)
Comment (Froffy025) [reply Resources/Prototypes/Traits/inconveniences.yml:32]: got it. keeping the prototypes for FrontalLisp and Snoring in disabilities was causing an error on startup, is it cool if i keep those removed from that file?

Comment (Froffy025) [reply Resources/Prototypes/Traits/inconveniences.yml:32]: not a critical error mind you it just got red and angry at me for duplicates

Comment (QuietlyWhisper) [reply Resources/Prototypes/Traits/inconveniences.yml]: Simply changing the ID should work like `CMFrontalLisp` on the CM file should work, do they not show up if you do that?

**Pull request closed without merging**